File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        String[] str = my_string.split(" ");

        for (int i = 0; i < str.length; i += 2) {
            int num = Integer.parseInt(str[i]);

            if (i > 0) {
                if (str[i - 1].equals("+")) {
                    answer += num;
                } else if (str[i - 1].equals("-")) {
                    answer -= num;
                }
            } else {
                answer = num;  
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.js
function solution(my_string) {
    let answer = 0;
        let str = my_string.split(" ");

        for (let i = 0; i < str.length; i += 2) {
            let num = parseInt(str[i]);

            if (i > 0) {
                if (str[i-1] === "+") {
                    answer += num;
                } else if (str[i-1] === "-") {
                    answer -= num;
                }
            } else {
                answer = num;  
            }
        }
        return answer;
}